### ISSUE TITLE
0.12.20 depr() takes from 1 to 2 positional arguments but 4 were given

### ISSUE THREAD

**Issue by ShaneHarvey** [bottlepy/bottle#1377]
I'm facing this error from bottle 0.12.20:

```
$ mongo-orchestration start --no-fork
Starting Mongo Orchestration on port 8889...
Bottle v0.12.20 server starting up (using CherryPyServer())...
Listening on http://localhost:8889/
Hit Ctrl-C to quit.

Traceback (most recent call last):
  File "/Users/shane/tmp/venv37/lib/python3.7/site-packages/mongo_orchestration/server.py", line 134, in run
    server=self.args.server)
  File "/Users/shane/tmp/venv37/bin/bottle.py", line 3166, in run
    server.run(app)
  File "/Users/shane/tmp/venv37/bin/bottle.py", line 2799, in run
    "project called 'cheroot'.", "Use the 'cheroot' server "
TypeError: depr() takes from 1 to 2 positional arguments but 4 were given
```

The [error is raised here](https://github.com/bottlepy/bottle/blob/0.12.20/bottle.py#L2799):
```python
class CherryPyServer(ServerAdapter):
    def run(self, handler): # pragma: no cover
        depr(0, 13, "The wsgi server part of cherrypy was split into a new "
                    "project called 'cheroot'.", "Use the 'cheroot' server "
                    "adapter instead of cherrypy.")
        from cherrypy import wsgiserver # This will fail for CherryPy >= 9
```

Where `depr` is defined as:
```python
def depr(message, hard=False):
    warnings.warn(message, DeprecationWarning, stacklevel=3)
```
https://github.com/bottlepy/bottle/blob/0.12.20/bottle.py#L150

**Comment by defnull** [comment]
So `cherrypy>9` is actually still in use? Interesting. I'll release a fix shortly.

**Comment by defnull** [comment]
Fixed in 0.12.21

**Comment by ShaneHarvey** [comment]
Thanks! I opened https://github.com/10gen/mongo-orchestration/issues/291 for us to upgrade from cherrypy<9. It looks like we pinned to <9 and never upgraded.
